WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml

```diff
@@ -64,6 +64,7 @@
                       VerticalScrollBarVisibility="{TemplateBinding (ScrollViewer.VerticalScrollBarVisibility)}">
                     <DockPanel>
                       <TextBlock Name="PART_ConversionWatermark"
+                                 Classes="watermark"
```

Comment:
It would make the selectors super long... I think this is a better approach than combining long selectors and then rider complains about it and disables text highlighting altogether. Which reduces maintainability.

Reply:
Sorry, I don't understand. I mean classes are reusable, but in this case, it makes no sense. You are using a class to modify a TextBlock.

So instead of this:
```
<Style Selector="c|CurrencyEntryBox:disabled /template/ TextBlock.watermark">
    <Setter Property="Foreground" Value="{DynamicResource TextControlPlaceholderForegroundDisabled}" />
  </Style>

  <Style Selector="c|CurrencyEntryBox:pointerover /template/ TextBlock.watermark">
    <Setter Property="Foreground" Value="{DynamicResource TextControlPlaceholderForegroundPointerOver}" />
  </Style>

  <Style Selector="c|CurrencyEntryBox:focus /template/ TextBlock.watermark">
    <Setter Property="Foreground" Value="{DynamicResource TextControlPlaceholderForegroundFocused}" />
  </Style>
```

This should be the usual approach:
```
  <Style Selector="c|CurrencyEntryBox:disabled /template/ TextBlock#PART_ConversionWatermark">
    <Setter Property="Foreground" Value="{DynamicResource TextControlPlaceholderForegroundDisabled}" />
  </Style>

  <Style Selector="c|CurrencyEntryBox:pointerover /template/ TextBlock#PART_ConversionWatermark">
    <Setter Property="Foreground" Value="{DynamicResource TextControlPlaceholderForegroundPointerOver}" />
  </Style>

  <Style Selector="c|CurrencyEntryBox:focus /template/ TextBlock#PART_ConversionWatermark">
    <Setter Property="Foreground" Value="{DynamicResource TextControlPlaceholderForegroundFocused}" />
  </Style>
```

Like it is with `PART_Watermark`.